docs/langref/src/concepts/layouting.md

```diff
@@ -46,10 +47,10 @@ export component Example inherits Window {
 }
 ```
 
-The position of both rectangles is fixed, as well as the size of the inner green one.
-The outer blue rectangle however has a size that's automatically calculated using binding
+The positions of both rectangles and the size of the inner green one are hard-coded.
```

Comment:
I think "fixed" is a much better counter part to variable than "hard-coded", a term that I think is primarily used for code, while this here is about layouts.

```suggestion
The positions of both rectangles and the size of the inner green one are fixed.
```